ApplicationLibCode/Application/RiaDefines.h

```diff
@@ -204,4 +204,23 @@ enum class RowCount
     ROWS_4 = 4,
 };
 
+enum class MultiPlotPageUpdateType : uint32_t
```

Comment:
Try to make it clearer that this is a bit field used, something along the lines of this, use binary literals
```
enum class Bits //C++14
{
 CLEAR = 0b0,
 SET =0b1
};
````